<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsolidateUnspentWizardSendPage</class>
 <widget class="QWizardPage" name="ConsolidateUnspentWizardSendPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>851</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="sendIntroLabel">
        <property name="text">
         <string>Step 3: Confirm Consolidation Transaction Details. Transaction will be ready to send when Finish is pressed.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="InputQuantityTextLabel">
        <property name="text">
         <string>Number of Inputs</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="InputQuantityLabel">
        <property name="text">
         <string>999999</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="feeTextLabel">
        <property name="text">
         <string>Transaction Fee</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="feeLabel">
        <property name="text">
         <string>99.9999</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="afterFeeAmountTextLabel">
        <property name="text">
         <string>Amount</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="afterFeeAmountLabel">
        <property name="text">
         <string>999999999.9999</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="destinationAddressTextLabel">
        <property name="text">
         <string>Destination Address</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="destinationAddressLabel">
        <property name="text">
         <string>address</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="destinationAddressLabelTextLabel">
        <property name="text">
         <string>Destination Address Label</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="destinationAddressLabelLabel">
        <property name="text">
         <string>label</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
